### PR TITLE
community: Make behaviour of CTRL more similar to Minecraft

### PR DESCRIPTION
**Pull request by Arta48**
i make CTRL working as in Minecraft

I mean when you sprinting and release CTRL, you stop sprinting. I fix it, sprinting stops when you stop pressing W. Also when you hold CTRL, press W, release W and press W again, sprinting doesn't work. It's also fixed. FOV increases when standing, etc. are fixed.

### CODE REVIEW

**Comment by obiwac** [comment]
Hey, thanks for the PR, could you elaborate on what you mean by this more specifically?

I'll review the patch a little later :)

**Comment by Arta48** [comment]
i mean when you sprinting and release CTRL, you stop sprinting. I fix it, sprinting stop when you stop pressing W. Also when you hold CTRL, press W, release W and press W again sprinting doesn't work. It's also fixed etc.

**Comment by Arta48** [comment]
I mean when you sprinting and release CTRL, you stop sprinting. I fix it, sprinting stops when you stop pressing W. Also when you hold CTRL, press W, release W and press W again, sprinting doesn't work. It's also fixed. FOV increases when standing, etc. are fixed.

**Comment by Arta48** [comment]
Шt probably went wrong

File: community/keyboard_mouse.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 import controller
 import math
 
+W_pressed = False
+LCTRL_pressed = False
```
Comment (obiwac) [inline community/keyboard_mouse.py:7]: Could these globals be added to the `Keyboard_Mouse` class instead?

**Review by obiwac — CHANGES_REQUESTED** [review]
Seems generally pretty good. Could you make your commit name a little more descriptive though? Also, if you could squash the two commits you made into one that would be great.

I assume you've used the GitHub web interface to do your modifications - I'd recommend using [git](https://git-scm.com) locally to make your changes, or, alternatively, [GitHub Desktop](https://desktop.github.com) or [GitKraken](https://gitkraken.com).

I can also commit in your place if you don't feel like doing it :)

File: community/keyboard_mouse.py (hunk 1 above)
Comment (Arta48) [reply community/keyboard_mouse.py:7]: I tried, it's not working

**Comment by Arta48** [comment]
> Seems generally pretty good. Could you make your commit name a little more descriptive though? Also, if you could squash the two commits you made into one that would be great.
> 
> I assume you've used the GitHub web interface to do your modifications - I'd recommend using [git](https://git-scm.com) locally to make your changes, or, alternatively, [GitHub Desktop](https://desktop.github.com) or [GitKraken](https://gitkraken.com).
> 
> I can also commit in your place if you don't feel like doing it :)

Yes, this is difficult for me

File: community/keyboard_mouse.py (hunk 1 above)
Comment (Arta48) [reply community/keyboard_mouse.py]: Already made two into one

**Comment by obiwac** [comment]
Committed :)

**Comment by Arta48** [comment]
Good luck :)

**Review by Arta48 — COMMENTED** [review]
1

**Pull request closed without merging**